apps/ui/src/Pages/RidesFeed/CreateRideDialog.tsx

```diff
@@ -65,6 +65,23 @@ const CreateRideDialog = ({
     setOpen(false);
   };
 
+  const handleGasSeatsChange = (fieldName: string, value: string) => {
+    switch (fieldName) {
+      case 'gasMoney':
+        setGasMoney(value);
+        break;
+      case 'seats':
+        setSeats(value);
+        break;
+      default:
+        break;
+    }
+  };
+  const isGasMoneyNegative = Number(gasMoney) < 0;
+  const isSeatsLessThanOne = Number(seats) < 1;
+  const isSeatsNotInteger = !Number.isInteger(Number(seats));
+
+
   const handleSubmit = async () => {
     if (!community || !startLocation || !destination || !gasMoney || !seats) {
       alert('All fields are required.');
```

Comment:
change the alert to write to const error and display the error in the dialog

Reply:
changed it to array of errors